keyvaluestore.js

```diff
@@ -6,14 +6,22 @@ app.use(bodyParser.json()); // support json encoded bodies
 app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
 
 const port = process.env.MICRO_KEYVALUESTORE_PORT || process.env.PORT || 9000;
+const masterEndpoint = process.env.MICRO_MASTER_ENDPOINT || '';
+const dbEndpoint = process.env.MICRO_DB_ENDPOINT || '';
+const storageEndpoint = process.env.MICRO_STORAGE_ENDPOINT || '';
 
 var keyValueStore = {}
 
 app.get('/:key/', (req, res) => {  
 	var key = req.params.key
 
+	v = mapKeyToService(key)
```

Comment:
Replace this with

```
if (key in keyValueStore) {
    res.send(keyValueStore[key])
} else {
    v = mapKeyToService(key);

    if (v) {
        console.log('Returning hard-coded service address: ' + v + ' ; key = ' + key);
        res.send(v);
    } else {
        res.status(404).send("Key not found")
    }
}
```

or even

```
if (key in keyValueStore) {
    res.send(keyValueStore[key])
    return;
}
```

And then continue with the rest, but without the `else` block.